test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval_long_expiration.test

```diff
@@ -72,7 +71,7 @@ payload='<?xml version="1.0"?>
     <attribute>temperature</attribute>
   </attributeList>
   <reference>http://127.0.0.1:'${LISTENER_PORT}'/notify</reference>
-  <duration>P99Y</duration>
+  <duration>PT11S</duration>
```

Comment:
The idea of the "_long_expiration.test" is to use a very very long duration. In fact, note the comment in the .test file: "#Subscribe for a LONG period of time (99 years - should give overflow in an int)".

Is this change to 11S really needed?